src/Backend/Modules/Blog/Domain/CommentRepository.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Backend\Modules\Blog\Domain;
+
+use Common\Locale;
+use Doctrine\DBAL\Connection;
+use Doctrine\ORM\EntityRepository;
+
+class CommentRepository extends EntityRepository
+{
+    public function listCountPerStatus(Locale $locale): array
+    {
+        $builder = $this->getEntityManager()
```

Comment:
```php
$builder = $this->createQueryBuilder('c')
    ->select('c.status, count(c.status) as number')
    ->where("c.locale = :locale")
    ->setParameter('locale', $locale)
    ->groupBy('c.status');
```

Reply:
Fixed in 0e741a4ea2a0b9c8807860f108b201c8ca99548b